src/webapp/contexts/app-context.ts

```diff
@@ -11,6 +13,7 @@ export const AppContext = React.createContext<AppContextState | null>(null);
 
 export function useAppContext() {
     const context = useContext(AppContext);
+    i18n.setDefaultNamespace("dhis2-skeleton-app");
```

Comment:
Forcing the namespace was a fix for existing applications, but for new applications we decided to add the namespace in the `t` method so we can remove this.

You can see more details in this [PR](https://github.com/EyeSeeTea/dhis2-app-skeleton/pull/90).